src/System.IO/src/project.json

```diff
@@ -59,7 +59,7 @@
     },
     "uap10.1": {
       "dependencies": {
-        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24410-00"
+        "Microsoft.TargetingPack.Private.NETNative": "1.0.1-beta-24415-00"
```

Comment:
Thanks, yep, I misunderstood the comment (my own fault, comment's fine). Changed back so these dependencies won't be touched.